docker-compose.cms.yml

```diff
@@ -4,6 +4,9 @@ services:
     build:
       context: https://github.com/USSF-ORBIT/ussf-portal-cms.git#main
       dockerfile: Dockerfile
+      target: e2e-local
```

Comment:
Does 'e2e-local' need to be hard coded here?

Reply:
Good question. I think it's okay here since we only ever use the `docker-compose.cms.yml` when running things locally in docker via `yarn cms:up`. If we started using this docker compose file elsewhere we would want to consider changing it like in USSF-ORBIT/ussf-portal#163. That PR is different since the E2E workflow uses docker compose to start up all the various services.